http-ballerina-tests/tests/service_cors_test.bal

```diff
@@ -295,7 +295,7 @@ function testPreFlightReqServiceResourceCorsOverride() {
     };
     http:Response|error response = corsClient->options("/hello1/test1", headers);
     if (response is http:Response) {
-        assertEqualsCorsResponse(response, 200, "http://www.wso2.com", "true", "X-PINGOTHER", HTTP_METHOD_POST, "-1");
+        assertEqualsCorsResponse(response, 204, "http://www.wso2.com", "true", "X-PINGOTHER", HTTP_METHOD_POST, "-1");
```

Comment:
Shall we use `http:STATUS_NO_CONTENT` constant instead of `204` ? Check for other places as well.

Reply:
Theses changes applies to 900+ tests cases. Shall we fix those in a separate refactoring PR?